std_msgs/Header header
uint32 sending_agent
LabeledObject[] objects
uint8[] data